public/locales/en.json

```
@@ -271,7 +271,7 @@
             },
             "lines": "{{lines}} Lines",
             "length": "Length {{length}}",
-            "lengthTooltip": "You can write up to 100000 characters in this document."
+            "lengthTooltip": "You still may write {{remaining}} characters until you reach the limit for this document."
```

```suggestion
            "lengthTooltip": "You may still write {{remaining}} characters until you reach the limit for this note."
```

Not sure on the may still part.

---

Although both variants should be acceptable, the phrase "you may still ..." seems to be better chosen. I'll change that.
The term "document" is taken from the wording in version 1.6. But as we're referring to "notes" everywhere it should probably be used here too.